src/Abp/Resources/Embedded/EmbeddedResourceManager.cs

```diff
@@ -36,8 +54,25 @@ public EmbeddedResourceItem GetResource(string fullPath)
             {
                 source.AddResources(resources);
             }
-
             return resources;
         }
+
+        class EmbeddedResourceItemComparer : IEqualityComparer<string>
+        {
+            public bool Equals(string fullPath1, string fullPath2)
+            {
+                return InvariantResourceName(fullPath1).Equals(InvariantResourceName(fullPath2));
+            }
+
+            public int GetHashCode(string fullPath)
+            {
+                return InvariantResourceName(fullPath).GetHashCode();
+            }
+            private string InvariantResourceName(string fullPath)
+            {
+                return fullPath.Replace("/", ".");
```

Comment:
@sachatrauwaen is it possible to use EmbeddedResourcePathHelper.NormalizePath here or is it a different case ?

Reply:
Hi source code is part of a EmbeddedResourceItemComparer class, witch is notg used. The whole inner class can be deleted

Reply:
Thanks @sachatrauwaen. I will remove it after the merge operation.

Reply:
Thanks @ismcagdas for this great framework !